### ISSUE TITLE
error decoding response body

### ISSUE THREAD

**Issue by GloftOfficial** [AlyoshaVasilieva/luminous-ttv#12]
Seems like twitch changed something , was working fine yesterday now I get response 

 `{"code":500,"error":"error decoding response body: invalid type: null, expected struct PlaybackAccessToken at line 1 column 126\n\nCaused by:\n    invalid type: null, expected struct PlaybackAccessToken at line 1 column 126"}`

Anyone having the same issue?

**Comment by 2bc4** [comment]
I can also reproduce on multiple different servers, not just luminous. Seemed to start happening around an hour ago.

**Comment by cabbagefork** [comment]
Can confirm I am also experiencing issues. Tried version 0.5.3 and 0.5.2, TTV LOL PRO (with luminous in the backend), and the latest extension but nothing works. Seems like Twitch did change something recently.

**Comment by AlyoshaVasilieva** [comment]
closing as dupe even though this issue was opened first (woops), see https://github.com/AlyoshaVasilieva/luminous-ttv/issues/13#issuecomment-1571229339

tldr shit's fucked
